### PR TITLE
Added tests for ReviewContainer and ReviewTile

### PR DESCRIPTION
**Pull request by nkim1225**

### CODE REVIEW

File: db/migrate/20190426200757_create_reviews.rb [hunk 1]
```diff
@@ -1,13 +0,0 @@
-class CreateReviews < ActiveRecord::Migration[5.2]
```
Comment (CassandraNewell) [inline db/migrate/20190426200757_create_reviews.rb]: Why was this file deleted?

File: spec/controllers/reviews_controller_spec.rb [hunk 2]
```diff
@@ -20,6 +20,7 @@
       prev_count = Review.count
       post(:create, params: {meme_id: meme_1.id, review: post_json})
 
+
```
Comment (CassandraNewell) [inline spec/controllers/reviews_controller_spec.rb]: hahahaha don't let Pat know you added an extra line out of nowhere! Please undo :)

File: spec/javascript/reviewTile.js [hunk 3]
```diff
@@ -2,30 +2,42 @@ import testHelper from "./testHelper";
...
 describe("ReviewTile", () => {
-  let wrapper,
-      timestamp,
-      username,
-      rating,
-      comment;
+  let id, meme_id, timestamp, username, rating, comment, wrapper;
 
   beforeEach(() => {
+    jasmineEnzyme();
```
Comment (CassandraNewell) [inline spec/javascript/reviewTile.js]: This is an unnecessary addition -- please remove

File: spec/javascript/reviewsContainer.js [hunk 4]
```diff
@@ -1,18 +1,50 @@
...
+    }
+  ];
 
   beforeEach(() => {
-    wrapper = mount(
-      <ReviewsContainer
-        memeId={3}
-      />
-    );
+    jasmineEnzyme();
```
Comment (CassandraNewell) [inline spec/javascript/reviewsContainer.js]: Unnecessary addition -- please remove

File: db/migrate/20190426200757_create_reviews.rb (hunk 1 above)
Comment (nkim1225) [reply db/migrate/20190426200757_create_reviews.rb]: we had 2 migrations when we merged for some reason so we had to delete one.

Comment (nkim1225) [reply db/migrate/20190426200757_create_reviews.rb]: nvm I see what you mean :D
